blocks/columns/columns.css

```diff
@@ -37,37 +54,89 @@
     margin: var(--spacer-layout-02) auto;
 }
 
+.section-top > div:first-child {
+    margin: 0 auto;
+    padding-top: var(--spacer-layout-04);
+    }
+
+.content-band div:empty {
+    display: none;
+    }
+
+.content-band.icon-list > div > div {
+    margin-top: var(--spacer-element-05);
+    margin-bottom: var(--spacer-element-06);
+    }
+
+.columns.content-band li {
+    list-style-image: url("../icons/arrow-purple.svg");
+    margin-bottom: var(--spacer-element-03);
+    line-height: var(--line-height-160);
+    }
+
+.section.leadership-content h3 {
+    margin: var(--spacer-element-08) auto var(--spacer-element-05) auto;
+    }
+
+.content-band.icon-list.picture-circle h3 {
+    margin: var(--spacer-layout-01) auto;
+    }
+
+.columns img {
+    width: 100%;
+    }
+
+.leadership-content img {
+    border-radius: 15px;
+    }
+
+.section-top-grid > div > div:nth-child(odd) img {
+    border-radius: 90% 0 0;
+    width: 230px;
+    height: 230px;
+    }
+
+.section-top-grid > div > div:nth-child(even) img {
+    border-radius: 0 90% 0 0;
+    width: 98px;
+    height: 95px;
+    max-width: 98px;
+    }
+
+.content-band.icon-list.picture-circle img {
+    border-radius: 50%;
+    height: 158px;
+    width: 158px;
+    }
+
```

Comment:
🤔 Hmm, there appear to be lots of selectors that aren't targeting `.columns`. Does `.content-band` always have to be added to a columns block? IMO, it would be easier to understand if all these selectors included `.columns` but maybe there are technical reasons why this is not possible.

Reply:
I'm not 100% sure, those classes pre-date me. I can add .columns though for consistency.

Reply:
actually there are a lot of things that changed with the conflict resolutions, etc, so I think we can revisit the global addition of .columns in a new ticket/branch.